include/wil/winrt.h

```diff
@@ -29,25 +29,9 @@
 #endif
 
 /// @cond
-#if defined(WIL_ENABLE_EXCEPTIONS) && !defined(__WI_HAS_STD_LESS)
-#define __WI_HAS_STD_LESS 1
-#if WI_HAS_INCLUDE(<functional>, 1) // Functional header available since C++11... fall back to assume present
+#if WIL_USE_STL
 #include <functional>
-#else
-// Fall back to the old way of forward declaring std::less

```

Comment:
So is this fallback so outdated that it simply isn't needed anymore?

Reply:
Pretty much. If we can determine that it's safe to include `<functional>` - and include it - then there's no need to forward declare it (which is a violation of the standard anyway).